ansible/roles/kong-api/defaults/main.yml

```diff
@@ -6552,3 +6559,599 @@ kong_apis:
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: startBotUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/bot/start"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/bot/start"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userAdmin'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: pauseBotUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/bot/pause"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/bot/pause"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userAdmin'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: searchBotUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/bot/search"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/bot/search"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userAdmin'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: deleteBotUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/bot/delete"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/bot/delete"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userAdmin'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: updateBotUCI
+    uris: "{{ uci_admin_prefix }}/admin/v1/bot/update"
+    upstream_url: "{{ uci_admin_service_url }}/admin/v1/bot/update"
+    strip_uri: true
+    plugins:
+    - name: jwt
+    - name: cors
+    - "{{ statsd_pulgin }}"
+    - name: acl
+      config.whitelist:
+        - 'userAdmin'
+    - name: rate-limiting
+      config.policy: local
+      config.hour: "{{ medium_rate_limit_per_hour }}"
+      config.limit_by: credential
+    - name: request-size-limiting
+      config.allowed_payload_size: "{{ small_request_size_limit }}"
+
+  - name: getAllUserForBotUCI
```

Comment:
Missing in master sheet

Reply:
Added.

Reply:
I am unable to see this API in master sheet. Please help me with the row number